### PR TITLE
Better handle iModel data errors and add message configurability

### PR DESCRIPTION
**Pull request by kckst8**

### CODE REVIEW

File: packages/modules/itwin-error-handling-react/src/components/fallback/FallbackUi.tsx [hunk 1]
```diff
@@ -36,7 +38,9 @@ export const FallbackUi = ({
           <Subheading2>{title}</Subheading2>
         </p>
         <p className={styles.message} data-testid="test-fallback-text">
-          <Subheading>{text}</Subheading>
+          <Subheading
+            dangerouslySetInnerHTML={{ __html: innerText }}
+          ></Subheading>
```
Comment (aruniverse) [inline packages/modules/itwin-error-handling-react/src/components/fallback/FallbackUi.tsx:43]: do you need the separate closing tag?

File: packages/modules/itwin-viewer-react/src/services/Initializer.ts [hunk 2]
```diff
@@ -105,11 +107,53 @@ class Initializer {
...
+  public static async getSynchronizerUrl(
+    contextId: string,
+    iModelId: string
+  ): Promise<string> {
+    if (!this._synchronizerRootUrl) {
+      const urlDiscoveryClient = new UrlDiscoveryClient();
+      this._synchronizerRootUrl = await urlDiscoveryClient.discoverUrl(
+        new ClientRequestContext(),
+        "itwinbridgeportal",
+        Config.App.get("imjs_buddi_resolve_url_using_region")
```
Comment (aruniverse) [inline packages/modules/itwin-viewer-react/src/services/Initializer.ts:119]: i think you can actually pass undefined here, which should default to using the value for imjs_buddi_resolve_url_using_region under the hood

**Review by aruniverse — APPROVED** [review]

File: packages/modules/itwin-error-handling-react/src/components/fallback/FallbackUi.tsx (hunk 1 above)
Comment (kckst8) [reply packages/modules/itwin-error-handling-react/src/components/fallback/FallbackUi.tsx:43]: shouldn't now that there's no children. I'll update

File: packages/modules/itwin-viewer-react/src/services/Initializer.ts (hunk 2 above)
Comment (kckst8) [reply packages/modules/itwin-viewer-react/src/services/Initializer.ts:119]: thanks. I'll update

**Pull request merged**